Pod/Classes/DependencyContainer.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DependencyContainer.swift
+//  Pods
+//
+//  Created by Raymond, Johnathan (ETW - FLEX Resource) on 3/7/17.
+//
+//
+
+import Foundation
+
+public struct DependencyContainer {
+    
+    fileprivate let dependencies: Dictionary<ObjectIdentifier, Injectable>
+    
+    internal init(dependencies: Dictionary<ObjectIdentifier, Injectable>) {
```

Comment:
Would be nice to have the option to key off of different things so that you could register multiple versions of the same dependency